flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlNodeToOperationConversion.java

```diff
@@ -304,6 +315,9 @@ private static Optional<Operation> convertValidatedSqlNode(
             }
             return Optional.of(
                     converter.createTableConverter.convertCreateTable((SqlCreateTable) validated));
+        } else if (validated instanceof SqlCreateModel) {
+            return Optional.of(
+                    converter.createModelConverter.convertCreateModel((SqlCreateModel) validated));
```

Comment:
 Is there anything that blocks from moving it to `SqlNodeConverters` instead of putting here?